telemetry/telemetry.go

```diff
@@ -0,0 +1,30 @@
+package telemetry
+
+import (
+	"fmt"
+
+	servertypes "github.com/cosmos/cosmos-sdk/server/types"
+	"github.com/mitchellh/mapstructure"
+)
+
+// globalTelemetryConfig is a private variable that stores cardinality level.
+// It is set on initialization and does not change for the lifetime of the program.
```

Comment:
TIP: Keep comments "ever green". Saying "stores cardinality level" is correct as of RIGHT NOW, but most people don't bother updating comments in the future.

Reply:
Changed the comment to its clear this is about the lifecycle of the process.